docs/source/actions/queue.rst

```diff
@@ -240,6 +242,11 @@ A ``queue_rules`` takes the following parameter:
      - list of ``conditions``
      -
      - The list of ``conditions`` to match to get the queued pull request merged.
+   * - ``speculative_checks``
+     - int
+     - 5
+     - The number of car embarked to merge in the merge train.
```

Comment:
As a user, if you're just skimming through the doc, there's 0 chance you understand car/train here.
```suggestion
     - The maximum number of checks to run in parallel in the queue.
```

I think it's still easy to understand. And if we ever have the batch-merge feature we can add

`merge_batch_size: maximum number of pull request to merge at the same time`